### PR TITLE
Add persistance volume policy

### PR DESCRIPTION
**Pull request by FColinet**

### CODE REVIEW

**Review by sgissi — COMMENTED** [review]
Thank you for the PR. I think it would be better to have a generic annotation map instead. Do you mind changing the PR for that? The end result for users would be something like this in values.yaml:

```
persistence:
  annotations:
    helm.sh/resource-policy: keep
    something.else/abc: true
```

File: vaultwarden/templates/pvc.yaml [hunk 1]
```diff
@@ -3,6 +3,10 @@ kind: PersistentVolumeClaim
 apiVersion: v1
 metadata:
   name: {{ include "vaultwarden.fullname" . }}
+  annotations:
+    {{- if .Values.persistence.resourcePolicy }}
+    helm.sh/resource-policy: {{ .Values.persistence.resourcePolicy | quote }}
```
[the remark this answers is not in the document]
Comment (FColinet) [reply vaultwarden/templates/pvc.yaml]: Hi, 
yes, you are right. i changed to a generic object.

**Comment by sgissi** [comment]
Look good, thanks for the update. I'll test this soon and release a new minor chart version.

**Comment by sgissi** [comment]
Ok, new chart 1.2.1 released with your updates. Thank you for the PR!

**Pull request merged**